docs/cloud/rbac/rbac.md

```diff
@@ -0,0 +1,77 @@
+# RBAC (Private Beta)
+
+Vault radar supports RBAC at the resource level, allowing admin users to manage the access of a group's permissions to individual resources.
+
+## Setup Overview
+1. [Create a group for the organization](#create-a-group-for-the-organization)
+2. [Add members to the group](#add-members-to-the-group)
+3. [Create a project level group](#create-a-project-level-group)
+4. [Assign a Vault Radar developer role to the group](#assign-a-vault-radar-developer-role-to-the-group)
+5. [Add Vault Radar resources to the project level group](#add-vault-radar-resources-to-a-project-level-group)
+
+## Additional guides
+- [Removing Vault Radar resources from a project level group](#removing-vault-radar-resources-from-a-project-level-group)
+- [Accessing a Vault Radar resource once RBAC has been applied](#accessing-a-vault-radar-resource-once-rbac-has-been-applied)
+- [Accessing a Vault Radar resource as a non-admin user without any RBAC applied](#accessing-a-vault-radar-resource-as-a-non-admin-user-without-any-rbac-applied)
+
+### Create a group for the organization
+1. Navigate to the organizations page within HCP and click on an organization ![](img/GROUP-RBAC-SETUP-1.png)
+2. Within the organization, navigate to the Access Control page ![](img/GROUP-RBAC-SETUP-2.png)
+3. Within the access control page for the organization, navigate to the Groups page ![](img/GROUP-RBAC-SETUP-3.png)
+4. Click create group ![](img/GROUP-RBAC-SETUP-4.png)
+5. Enter a name for the group and submit ![](img/GROUP-RBAC-SETUP-5.png)
+
+---
+
+### Add members to the group
+**In order for the assigned group member to have RBAC enforced, the user must not have any role assigned role at the organization level**
```

Comment:
must not have any role assigned role at the organization level **or project level**.